
# Xingpeng Jiang 

### Ph.D,  Professor, and Chair
### School of Computer, Central China Normal University, Wuhan, Hubei Province, China


## Positions
### Professor, School of Computer, Central China Normal University, China, 2015-present
### Postdoc  Scientist,College of Computing and Informatics, Drexel University, US, 2012-2015
### Postdoc Associate, Department of Biology, McMaster University, Canada, 2009-2012

## Educations
### Ph.D., Institute of Automation,  China Academy of Sciences, 2005-2009
### Master, School of Applied Science, Beijing University of Technology, 2001-2004,
### Bachelor, School of Mathematics &Statistics, Qufu Normal University, 1997-2001 

## Grants and Honours
### 1.“Chutian Young Scholar” Talent Program in Hubei Province, China, 2015-2019 (250,000 RMB)
### 2.co-PI, Pattern extraction and analysis from high-throughput microbiome data, key program of National Science Funding of China (61532008), 2016-2020 (2900,000 RMB)
### 3.PI,  Computational prediction of high-order microbial interaction from multi-omics data, general program of National Science Funding of China, 2019-2022 (600,000 RMB)

## Academic service
### 1.Associate Editor of Journal of Artificial Intelligence for Medical Sciences, since 2021
### 2.Editor board of Advances in Biomarker Sciences and Technology, since 2019
### 3.Program Committee Member of IEEE International Conference on Bioinformatics and Biomedicine  since 2013
### 4.Committee Member of the Bioinformatics Committee of the Chinese Computer Society, since 2017
### 5.Standing Committee Member of Gene Detection Technology Branch of the Chinese Pharmaceutical Biotechnology Association, since 2018
### 6.Vice Chair of the Hubei Province Bioinformatics Society, China, since 2018
### 7.Editor board of International Journal of Data Mining and Bioinformatics, since 2014.  
### 8.Guest Editorial for Special Section on BIBM. IEEE/ACM Transactions on Computational Biology and Bioinformatics, 2013

## Selected Publications
### Sun H, Huang X, Huo B, Tan Y,  He T, Jiang X*. Detecting sparse microbial association signals adaptively from longitudinal microbiome data based on generalized estimating equations. Briefs in Bioinformatics. 2022(Accepted, DOI: 10.1093/bib/bbac149)
### Sun H, Huang X, Fu L, Huo B, He T, Jiang X*. A powerful adaptive microbiome-based association test for microbial association signals with diverse sparsity levels. Journal of Genetics and Genomics. 2021;48(9):851-9
### Zhu Q, Li B, He T, Li G, Jiang X*: Robust biomarker discovery for microbiome-wide association studies. Methods 2020, 173:44-51. (SCI)
### Zhu Q, Huo B, Sun H, Li B, Jiang X*: Application of Deep Learning in Microbiome. Journal of Artificial Intelligence for Medical Sciences 2020, 1(1-2):23-29. (Invited Review)
### Ma Y, He T, Jiang X*: Multi‐network logistic matrix factorization for metabolite–disease interaction prediction. FEBS letters 2020, 594 (11), 1675-1684. (SCI)
### Zhu Q, Jiang X*, Zhu Q, Pan M, He T: Graph embedding deep learning guide microbial biomarkers' identification. Frontiers in Genetics 2019, 10:1182. (SCI)
### Ma Y, He T, Jiang X*: Projection-based neighborhood non-negative matrix factorization for lncRNA-protein interaction prediction. Frontiers in Genetics 2019, 10:1148. (SCI)
### Ma Y, He T, Ge L, Zhang C, Jiang X*: MiRNA-disease interaction prediction based on kernel neighborhood similarity and multi-network bidirectional propagation. BMC Medical Genomics 2019, 12(10):1-14. (SCI)
### Liu D, Ma Y, Jiang X*, He T: Predicting virus-host association by Kernelized logistic matrix factorization and similarity network fusion. BMC Bioinformatics 2019, 20(16):594. (SCI)
### Li X, Fu C, Zhong R, Zhong D, He T, Jiang X*: A hybrid deep learning framework for bacterial named entity recognition with domain features. BMC Bioinformatics 2019, 20(16):1-9. (SCI)
### Wang X, Li Y, He T, Jiang X*, Hu X: Recognition of bacteria named entity using conditional random fields in spark. BMC Systems Biology 2018, 12(6):106. (SCI,)
### Jiang X, Hu X, Xu W: Microbiome Data Representation by Joint Nonnegative Matrix Factorization with Laplacian Regularization. IEEE/ACM Transactions on Computational Biology and Bioinformatics 2017, 14(2):353-359. (SCI)
### Ma Y, Hu X, He T, Jiang X*: Hessian regularization based symmetric nonnegative matrix factorization for clustering gene expression and microbiome data. Methods 2016, 111:80-84. (SCI)
### Jiang X, Hu X, He T: Identification of the clustering structure in microbiome data by density clustering on the Manhattan distance. Science China-Information Sciences 2016, 59(7):070104.  (SCI)
### Zhang Y, Hu X, Jiang X*: Multi-view clustering of microbiome samples by robust similarity network fusion and spectral clustering. IEEE/ACM Transactions on Computational Biology and Bioinformatics 2015, 14(2):264-271. (SCI)
### Xu W, Jiang X*, Hu X, Li G: Visualization of genetic disease-phenotype similarities by multiple maps t-SNE with Laplacian regularization. BMC Medical Genomics 2014, 7(2):1-9. (SCI)
### Jiang X, Xu W, Park E, Li G: Selecting Protein Families for Environmental Features Based on Manifold Regularization. IEEE Transactions on Nanobioscience 2014, 13(2):104-108. (SCI)
### Jiang X, Hu XT, Xu W, Park EK: Predicting Microbial Interactions using Vector Autoregressive Model with Graph Regularization. IEEE/ACM Transactions on Computational Biology and Bioinformatics 2014. (SCI)
### Jiang X, Hu X: Inferring microbial interaction networks based on consensus similarity network fusion. Science China-Life Sciences 2014, 57(11):1115-1120. (SCI)
### Jiang X, Hu X, Xu W, He T, Park EK: Comparison of dimensional reduction methods for detecting and visualizing novel patterns in human and marine microbiome. IEEE Transactions on Nanobioscience 2013, 12(3):199-205. (SCI)
### Bartram A, Jiang X, Lynch M, Masella A, Nicol G, Dushoff J, Neufeld J: Exploring links between pH and bacterial community composition in soils from the Craibstone Experimental Farm. FEMS Microbiology Ecology 2013, 87(2):403-415. (SCI)
### Jiang X, Weitz JS, Dushoff J: A non-negative matrix factorization framework for identifying modular patterns in metagenomic profile data. Journal of Mathematical Biology 2012, 64(4):697-711. (SCI)
### Jiang X, Langille MGI, Neches RY, Elliot M, Levin SA, Eisen JA, Weitz JS, Dushoff J: Functional Biogeography of Ocean Microbes Revealed through Non-Negative Matrix Factorization. PLoS ONE 2012, 7(9):e43866. (SCI)
### Jiang X, Liu B, Jiang J, Zhao H, Fan M, Zhang J, Fan Z, Jiang T: Modularity in the genetic disease-phenotype network. FEBS letters 2008, 582(17):2549-2554. (SCI)


